README.md

```diff
@@ -24,13 +24,15 @@ The rest of this document will focus on how this version of XCTest differs from
 
 ## Working on XCTest
 
+### On Linux
+
 XCTest can be built as part of the overall Swift package. When following [the instructions for building Swift](http://www.github.com/apple/swift), pass the `--xctest` option to the build script:
 
 ```sh
 swift/utils/build-script --xctest
 ```
 
-If you want to build just XCTest, use the `build_script.py` script at the root of the project. The `master` version of XCTest must be built with the `master` version of Swift.
+If you want to build just XCTest, use the `build_script.py` script at the root of the project. The `master` version of XCTest must be built with the `master` version of Swift. XCTest has a dependency upon Foundation, so you must have built the `master` version of that as well.
```

Comment:
I'm wondering how true it is that a `master` build of Foundation is necessary? I just hacked around a little bit and saw a successful build and test run using the Foundation+CoreFoundation shipped with the latest snapshot. Clearly there may be times where we depend on code only present on Foundation `master` but I would imagine that won't be too frequent.

Do you think it would be reasonable to treat `--foundation-build-dir` as an optional parameter to the build script, allowing the compiler to find a pre-installed Foundation library from a snapshot? This would simplify the use case where someone just wants to verify that their changes don't break `XCTest`'s Linux test suite without doing too much extra legwork getting a more complete swift.org development environment set up. The same logic applies to the `--swiftc` argument too, I think.

It'd be pretty neat if people could spin up an environment from a Docker image like [this](https://hub.docker.com/r/swiftdocker/swift/), clone just `swift-corelibs-xctest` and run `./build_script.py --test` without having to wait for `swift` and `llvm` to build which (AFAIK?) is currently required when building swift-corelibs-foundation from source.